### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Klishchar-Ihor**
- [DEMO LINK](https://Klishchar-Ihor.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Klishchar-Ihor.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/_card.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
+.cards-container {
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/_card.scss:1]: each blocke should be in separate file.
move styles for container in other file.

File: src/styles/blocks/_card.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    margin-bottom: 73px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  .card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/_card.scss:19]: follow to BEM rules block shouldn't have margins.
move margins to `.cards-container__card` element

File: src/styles/blocks/_header.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      &:hover {
+        color: var(--hover-color);
+      }
+
+      &--active {
+        color: var(--hover-color);
+        position: relative;
+
+        &::after {
+          content: " ";
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/_header.scss:51]:
```suggestion
          content: "";
```

File: src/styles/blocks/_stars.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background: url("../images/star.svg") no-repeat center;
+
+    .stars--4 &:nth-child(-n+4) {
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/_stars.scss:10]: styles for stars should be for any amount of stars
and pay attention on this point from [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
Screenshot from 2023-05-11 13-47-25

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by vlad-demchuk — APPROVED** [review]

**Pull request closed without merging**